libafl/src/executors/forkserver.rs

```diff
@@ -743,7 +765,7 @@ where
 
     /// Execute input, but side-step the execution counter.
     #[inline]
-    fn execute_input_uncounted(&mut self, input: &[u8]) -> Result<ExitKind, Error> {
+    pub fn execute_input_uncounted(&mut self, input: &[u8]) -> Result<ExitKind, Error> {
```

Comment:
This allows `ForkserverExecutor` to be reused independently without `Fuzzer`,  `EventManager` and `State`.